lib/brainstem/api_docs/builder.rb

```diff
@@ -0,0 +1,165 @@
+require 'brainstem/api_docs/introspectors/rails_introspector'
+require 'brainstem/api_docs/atlas'
+require 'active_support/core_ext/hash/slice'
+
+# TODO: Clean up public/private in this class.
```

Comment:
I should say-- please search this PR for "TODO" and identify any that you feel are important to have in a V1.